src/styles/tailwindcss.css

```diff
@@ -0,0 +1,9 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .btn {
+    @apply px-4 py-2 bg-blue-600 text-white rounded;
+  }
+}
```

Comment:
As before: we're not gonna use Tailwind (at least, not now).

Reply:
OK, you are right.